src/main/java/ConsoleMessage.java

```diff
@@ -1,10 +0,0 @@
-public class ConsoleMessage extends Message{
```

Comment:
eee, wszystkie zadania finalnie powinny zostać w tym repo :) 
Wrzuć je spowrotem, po prostu do jakieogś osobnego pakietu np /zad3/ (czy które tam to było)